openmc/geometry.py

```diff
@@ -1,11 +1,13 @@
+from __future__ import annotations
 import os
 import typing
 from collections import OrderedDict, defaultdict
 from collections.abc import Iterable
-from copy import deepcopy
```

Comment:
Oops, looks like I misled you. This is actually used; sorry about that! Can  you add this line back?